bin/oref0-ns-loop.sh

```diff
@@ -92,8 +92,13 @@ function glucose_fresh {
 }
 
 function find_valid_ns_glucose {
-    # TODO: use jq for this if possible
-    cat cgm/ns-glucose.json | json -c "minAgo=(new Date()-new Date(this.dateString))/60/1000; return minAgo < 10 && minAgo > -5 && this.glucose > 38"
+    dir_name=~/test_data/find-glucose$(date +"%Y-%m-%d-%T")
+    echo dir_name = $dir_name
+    mkdir -p $dir_name
+    cp cgm/ns-glucose.json  $dir_name
+    date --iso-8601=seconds >  $dir_name/date_string
+
+    run_remote_command 'json -f cgm/ns-glucose.json -c "minAgo=(new Date()-new Date(this.dateString))/60/1000; return minAgo < 10 && minAgo > -5 && this.glucose > 38"'
```

Comment:
Should this be using $dir_name/ns-glucose.json instead of cgm/ns-glucose.json?

As it is, I get:
```
Found recent valid BG:
dir_name = /root/test_data/find-glucose2020-02-14-19:46:10 [ { "_id": "5e476842403a6cc03ad98637", "device": "removed", "date": 1581738040315, "dateString": "2020-02-15T03:40:40.315Z", "sgv": 136, "direction": "FortyFiveDown", "type": "sgv", "filtered": 189, "unfiltered": 176, "rssi": -97, "noise": 1, "trend": 5, "state": "OK", "status": "OK", "glucose": 136 } ] (NOT VALID JSON: parse error: Invalid numeric literal at line 1, column 9)
```